<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/clear.png</normaloff>../images/clear.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Scan Range</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="startIp">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Start IP Address</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true">192.168.0.0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="endIp">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>End IP Address</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true">192.168.0.255</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnScan">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Start Range Scan (F5) / Stop (F7)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Custom Scan Range</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="stringIp">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Scan ranges of IP addresses. Can scan multiple ranges. For example: 192.168.0.1-110, 192.168.1.5-15</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>192.168.0.0-255</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCustomScan">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Start Custom Scan (F6) / Stop (F7)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Scan Configuration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Timeout for checking hostname (s) [-1 == 5s]</string>
           </property>
           <property name="text">
            <string>Hostname Timeout:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinHostTimeout">
           <property name="toolTip">
            <string>Timeout for checking hostname (s) [-1 == 5s]</string>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxHostname">
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxMac">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>MAC Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxManufacturer">
           <property name="text">
            <string>Manufacturer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxResults">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBoxAlive">
             <property name="text">
              <string>Show Alive Only</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btnClearIpList">
             <property name="toolTip">
              <string>Clear Scan Results (Stops Scan)</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="ipList">
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuClear_Scan_History">
     <property name="title">
      <string>Clear Scan History</string>
     </property>
     <addaction name="actionAll"/>
     <addaction name="separator"/>
     <addaction name="actionScan_Range"/>
     <addaction name="actionScan_Custom_History"/>
    </widget>
    <addaction name="menuClear_Scan_History"/>
    <addaction name="separator"/>
    <addaction name="actionSettings_4"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Range_Scan"/>
    <addaction name="actionStart_Stop_Scan"/>
    <addaction name="actionStop_Scan"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_For_Updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClean_2">
   <property name="text">
    <string>Clean</string>
   </property>
  </action>
  <action name="actionDist_Clean">
   <property name="text">
    <string>Dist Clean</string>
   </property>
  </action>
  <action name="actionSettings_2">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionSubmit_Issue">
   <property name="text">
    <string>Submit Issue</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDeploy">
   <property name="text">
    <string>Deploy</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionClear_Recent_Files">
   <property name="text">
    <string>Clear Recent Files</string>
   </property>
  </action>
  <action name="actionSettings_3">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSettings_4">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionServe">
   <property name="text">
    <string>Serve</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionScan_Range">
   <property name="text">
    <string>Scan Range</string>
   </property>
  </action>
  <action name="actionScan_Custom_History">
   <property name="text">
    <string>Custom Scan</string>
   </property>
  </action>
  <action name="actionStart_Range_Scan">
   <property name="text">
    <string>Start Range Scan</string>
   </property>
   <property name="toolTip">
    <string>Start Range Scan (F5) / Stop (F7)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStart_Stop_Scan">
   <property name="text">
    <string>Start Custom Scan</string>
   </property>
   <property name="toolTip">
    <string>Start Custom Scan (F6) / Stop (F7)</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStop_Scan">
   <property name="text">
    <string>Stop Scan</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionCheck_For_Updates">
   <property name="text">
    <string>Check For Updates</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenubar(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHostTimeout</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCustomScan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onScanCustom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>730</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnScan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>648</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAlive</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onResultsFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>737</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearIpList</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClearResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>checkBoxStateChanged(int,QString)</slot>
  <slot>onMenubar(QAction*)</slot>
  <slot>onEnvPathEdit(QString)</slot>
  <slot>updateBuildPaths()</slot>
  <slot>refreshPreview()</slot>
  <slot>refreshSystemDetails()</slot>
  <slot>saveConfig()</slot>
  <slot>onScanCustom()</slot>
  <slot>onScan()</slot>
  <slot>clearIpListItems()</slot>
  <slot>onResultsFilter()</slot>
  <slot>onClearResults()</slot>
 </slots>
</ui>
